lib/adzerk/util.rb

```diff
@@ -19,7 +19,7 @@ def uncamelize_data(data)
       # stop condition for the recursion
       return data unless data.respond_to?(:reduce)
       data.reduce({}) do |acc, (key, val)|
-        acc[key.underscore.to_sym] = case val
+        acc[/[a-zA-Z]/.match(key) ? key.underscore.to_sym : key] = case val
```

Comment:
Instead of `match` which returns a match object or `nil` as the response, you can use `.match?(key)` which will ensure a boolean response.

Reply:
Updated